diff --git a/customization.cfg b/customization.cfg
index f1b2cff..9c75807 100644
--- a/customization.cfg
+++ b/customization.cfg
@@ -25,7 +25,7 @@ _local_package_storing="false"
 _lib32="true"
 _opencl="true"
 _nvsettings="true"
-_eglwayland="true"
+_eglwayland="false"
 
 # Either "vulkandev" for vulkan developer drivers or "regular" for all others
 _driver_branch=""
@@ -36,7 +36,7 @@ _driver_version=""
 
 # Set to "true" to build open source modules if available, or leave empty to get a prompt
 # Available for 515.43.04 beta or newer only
-_open_source_modules=""
+_open_source_modules="false"
 
 # Set to "true" to blacklist nouveau. may be needed early kms on nvidia.
 # if you on like prime system, and not need nvidia module on startup, then can false.
@@ -49,7 +49,7 @@ _blacklist_nouveau=""
 _only_update_if_newer=""
 
 # Set to "true" to use DKMS or "false" to use regular modules. You can also use "full" to build both dkms and regular packages (don't use it if you don't know you need it!).
-_dkms=""
+_dkms="false"
 
 
 ## LEGACY OPTIONS
